docs/content/preview/admin/yb-admin.md

```diff
@@ -1682,7 +1682,32 @@ For example:
     create_change_data_stream ysql.yugabyte
 ```
 
+#### Creating a stream for Transactional CDC
 
+Creates a change data capture (CDC) DB stream for the specified namespace that can be used for Transactional CDC.
+This feature is {{<badge/tp>}}. The `yb_enable_cdc_consistent_snapshot_streams` flag can be used to enable the feature.
```

Comment:
```suggestion
This feature is {{<badge/tp>}}. Use the [yb_enable_cdc_consistent_snapshot_streams](../../reference/configuration/yb-tserver/#yb-enable-cdc-consistent-snapshot-streams) flag to enable the feature.

```

Reply:
Yes, this feature is in 2.20.2. Will backport these changes to stable pages as well